demo/src/vendor.ts

```diff
@@ -0,0 +1,6 @@
+import 'angular2/bundles/angular2-polyfills';
+
+import 'angular2/platform/browser';
+import 'angular2/core';
+import 'angular2/http';
+import 'angular2/router';
```

Comment:
The thing is that angular imports them anyway and it ends on the app bundle. By importing them by hand, they end on the vendor bundle.

TL;DR I can remove those two but will be on the bundle anyway. I will need that for the demo tho.